qa/sys/redhat/user_bootstrap.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+VERSION=`cat /vagrant/config/platforms.json | grep  latest | cut -d":" -f2 | sed 's/["\|,| ]//g'`
```

Comment:
Question is how? In pure bash this is the more generic way i could think
of, improvements welcome for sure!

On Thu, 12 May 2016 18:07 Tal Levy, <notifications@github.com> wrote:

> In qa/sys/redhat/user_bootstrap.sh
> <https://github.com/elastic/logstash/pull/5258#discussion_r63048123>:
>
> > @@ -0,0 +1,5 @@
> > +#!/usr/bin/env bash
> > +
> > +VERSION=`cat /vagrant/config/platforms.json | grep  latest | cut -d":" -f2 | sed 's/["\|,| ]//g'`
>
> can we make this less error prone and more explicit?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/logstash/pull/5258/files/476bea80018a39d0cfd62909455fa6626d7dcf68#r63048123>
>